modules/init-snippet-attach-ebs-volume/snippet.tpl

```diff
@@ -1,13 +1,5 @@
-# start snippet - attach EBS volume
-${init_prefix}
 export AWS_DEFAULT_REGION=${region}
 VOLUME_ID=${volume_id}
-if which wget; then
-  INSTANCE_ID="$(wget -O- http://169.254.169.254/latest/meta-data/instance-id)"
-elif which curl; then
-  INSTANCE_ID="$(curl http://169.254.169.254/latest/meta-data/instance-id)"
-fi
-
 if [ "x$${INSTANCE_ID}" == "x" ]; then
   echo 'OS not functioning'
```

Comment:
What would be "not functioning" here? Let's be more specific with the error message so an operator gets a more clear suggestion of what to do.

Reply:
OK.